dot/state/grandpa.go

```diff
@@ -228,8 +236,12 @@ func (s *GrandpaState) ApplyForcedChanges(importedBlockHeader *types.Header) err
 
 	logger.Debugf("applying forced change: %s", forcedChange)
 
-	// TODO: send the telemetry messages here
-	// afg.applying_forced_authority_set_change
+	efectiveNumberString := strconv.FormatUint(uint64(forcedChange.effectiveNumber()), 10)
+	s.blockState.telemetry.SendMessage(telemetry.NewAfgApplyingForcedAuthoritySetChange(
+		//TODO: check if this is correct
+		//https://github.com/paritytech/substrate/blob/50de15d8740a129db9c18a6698fbd183b00326a2/client/consensus/grandpa/src/authorities.rs#L501
+		efectiveNumberString,
```

Comment:
@EclesioMeloJunior Substrate is using "canon_height", I'm trying to figure out if this is the same that the efectiveNumber, can you confirm it if you know more about it?

Reply:
This is the `bestBlockNumber` info

Reply:
Thks!

Reply:
Done